guides/Makefile

```diff
@@ -21,3 +21,6 @@ clean:
 
 linkchecker:
 	find build -type f -name index\*html | xargs linkchecker -r1 -f common/linkchecker.ini --check-extern
+
+linkchecker:
```

Comment:
This target name seems wrong?

Reply:
Copy paste gone wrong :(

Having said that: how is this passing:
```
Run make linkchecker-tryer
Makefile:26: warning: overriding recipe for target 'linkchecker'
Makefile:23: warning: ignoring old recipe for target 'linkchecker'
make: Nothing to be done for 'linkchecker-tryer'.
```

Is that because it's a phony target?

Reply:
Probably?